definition/data/sheets/ComplexTypes.json

```diff
@@ -220,6 +220,13 @@
   {"ID": "Applicant", "ListElementCode": "representativeOrganisationDxAddress", "FieldType": "Text", "ElementLabel": "Organisation DxAddress", "SecurityClassification": "Public"},
   {"ID": "Applicant", "ListElementCode": "representativeOrganisationEmail", "FieldType": "Email", "ElementLabel": "Organisation Email", "SecurityClassification": "Public"},
   {"ID": "Applicant", "ListElementCode": "leadApplicantIndicator", "FieldType": "YesOrNo", "ElementLabel": "Lead applicant", "SecurityClassification": "Public"},
+  {"ID": "OtherDirection", "ListElementCode": "extraOrderDirection", "FieldType": "FixedList", "FieldTypeParameter": "ExtraOrderDirectionType", "ElementLabel": "Select direction", "SecurityClassification": "Public"},
+  {"ID": "OtherDirection", "ListElementCode": "extraDocUploadList", "FieldType": "Collection", "FieldTypeParameter": "Text", "ElementLabel": "Extra statements for document upload", "SecurityClassification": "Public", "FieldShowCondition": "extraOrderDirection=\"DOCUMENTS\""},
+  {"ID": "OtherDirection", "ListElementCode": "expertReports", "FieldType": "Collection", "FieldTypeParameter": "Text", "ElementLabel": "Extra statements for expert report", "SecurityClassification": "Public", "FieldShowCondition": "extraOrderDirection=\"EXPERT_REPORT_PERMISSION\""},
+  {"ID": "OtherDirection", "ListElementCode": "directionComment", "FieldType": "TextArea", "ElementLabel": "Extra direction", "SecurityClassification": "Public", "FieldShowCondition": "extraOrderDirection=\"OTHER\""},
+  {"ID": "OtherDirection", "ListElementCode": "sendBy", "FieldType": "Date", "ElementLabel": "Response deadline", "SecurityClassification": "Public"},
+  {"ID": "OtherDirection", "ListElementCode": "forParty", "FieldType": "FixedList", "FieldTypeParameter": "DirectionPartyType", "ElementLabel": "Direction is for", "SecurityClassification": "Public"},
+  {"ID": "OtherDirection", "ListElementCode": "otherDirectionHeaders", "FieldType": "FixedRadioList", "FieldTypeParameter": "OtherDirectionHeaderType", "ElementLabel": "Extra direction", "SecurityClassification": "Public", "FieldShowCondition": "extraOrderDirection=\"OTHER\""},
```

Comment:
Please move the line above 'directionComment',otherwise it will be in wrong order

Reply:
👍